Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -87,10 +87,7 @@ def get_normalize_by_bin_width(workspace, axes, **kwargs):
                 [artist[0].is_normalized for artist in current_artists])
             normalization = current_normalization
         else:
-            if mantid.kernel.config['graph1d.autodistribution'].lower() == 'on':
-                normalization = True
-            else:
-                normalization, _ = check_resample_to_regular_grid(workspace, **kwargs)
```

Comment:
Can I just check what happens in the case of irregular grids now?

Reply:
If the setting is off it wont normalize them by default but it can still be change in the individual plot - this is the same as it was in 4.2

Reply:
Ok. Thanks.